src/python/pants/option/global_options.py

```diff
@@ -1417,6 +1417,19 @@ def register_options(cls, register):
                 "when run with Docker."
             ),
         )
+        register(
+            "--streaming-workunits-level",
+            advanced=True,
+            type=LogLevel,
+            default=LogLevel.DEBUG,
+            help=(
+                "The Level of workunits that will be reported to streaming workunit event "
+                "receivers.\n\n"
+                "The filtering of workunits that occurs at higher levels is tree-aware: parent "
+                "pointers are fixed up when workunits are filtered such that the resulting workunits "
+                "always form a tree."
+            ),
```

Comment:
Eh, what..?
It is not at all clear to me what this tree thing implies.

Reply:
Yeah I'm also not sure what this is after? Maybe an example would help?